ipfs-cluster-service/main.go

```diff
@@ -231,6 +233,57 @@ configuration.
 			Usage:  "run the IPFS Cluster peer (default)",
 			Action: daemon,
 		},
+		{
+			Name:   "migration",
```

Comment:
migrate? upgrade? or `state upgrade`? Perhaps we should leave room for other state commands (`state inspect`, `state clean`, `state downgrade`, `state export`)...

Reply:
I like it!